static/js/pages/applications/PaymentHistoryPage_test.js

```diff
@@ -91,6 +92,19 @@ describe("PaymentHistoryPage", () => {
     })
   })
 
+  it("does not render the payment method if it is blank", async () => {
+    const nullPaymentMethodOrder = {
+      ...generateLegacyOrderPartial(),
+      total_price_paid: 10,
+      payment_method:   ""
+    }
+
+    fakeApplicationDetail.orders = [nullPaymentMethodOrder]
+
+    const { wrapper } = await renderPage()
+    assert.isNotOk(wrapper.find(".payment-method").exists())
+  })
+
```

Comment:
Can you add a test checking that the correct label is on the page if there is a refund?